lib/sql-string.js

```diff
@@ -193,14 +193,16 @@ SqlString.bufferToString = function(buffer, dialect) {
 };
 
 SqlString.objectToValues = function(object, timeZone) {
-  var values = [];
-  for (var key in object) {
```

Comment:
For in loops are really slow and - if I recall that correct - in some circumstances v8 will deopt because of the presence of those.
I agree that they are not as nice to read. In generell that's worth it though. But there is another solution you might prefer: 
Using
```js
var keys = Object.keys (something);
while (var name = keys.pop ()) {}
```
That seems to be even faster than normal for loops and you have the readability back :)